src/parser/expression.js

```diff
@@ -627,11 +622,11 @@ export default class ExpressionParser extends LValParser {
         return this.finishNode(node, "Super");
 
       case tt._import:
-        if (this.hasPlugin("importMeta") && this.lookahead().type === tt.dot) {
+        if (this.lookahead().type === tt.dot) {
           return this.parseImportMetaProperty();
```

Comment:
Yeah this problem was brought up by @motiz88, @danez in https://github.com/babel/babylon/pull/178/files#r87556780

If we make the error here I guess it's a different error if they just have `import.` vs `import.asdf` but maybe we shouldn't make a big deal out of it?